src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagJobs.java

```diff
@@ -0,0 +1,46 @@
+package org.opentripplanner.middleware.bugsnag;
+
+import org.opentripplanner.middleware.bugsnag.jobs.BugsnagEventJob;
+import org.opentripplanner.middleware.bugsnag.jobs.BugsnagEventRequestJob;
+import org.opentripplanner.middleware.utils.Scheduler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.opentripplanner.middleware.spark.Main.getConfigPropertyAsInt;
+
+/**
+ * Schedule the Bugsnag jobs
+ */
+public class BugsnagJobs {
+
+    private static final int BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_MINUTES
+        = getConfigPropertyAsInt("BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_MINUTES", 5);
+
+    private static final int BUGSNAG_EVENT_JOB_DELAY_IN_MINUTES
+        = getConfigPropertyAsInt("BUGSNAG_EVENT_JOB_DELAY_IN_MINUTES", 1);
+    private static final Logger LOG = LoggerFactory.getLogger(BugsnagReporter.class);
+
+    /**
+     * Define the frequency of each Bugsnag job and schedule
+     */
+    public static void initialize() {
+        BugsnagEventRequestJob bugsnagEventRequestJob = new BugsnagEventRequestJob();
+
+        Scheduler.scheduleJob(bugsnagEventRequestJob, 0,
```

Comment:
It's probably fine for now to schedule this in the same process as the server, but eventually, we should try to have separate processes for things that can be run independently.